.hgignore

```diff
@@ -1,2 +1,11 @@
+syntax: glob
```

Comment:
This repository is using Git, so changing `.hgignore` doesn't make much sense. I don't know why this file is still here, perhaps @jaraco is using it via hg-git or something like that.